app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -73,7 +76,11 @@ def updated_at
         @model.updated_at.to_fs(:us)
       end
 
-      def publish_messages
+      def publish_messages_on_creation
+        Emq::Publisher.publish(@model.primary_aliquot, Pipelines.pacbio, 'volume_tracking')
+      end
+
+      def publish_messages_on_update
         Messages.publish(@model.sequencing_runs, Pipelines.pacbio.message)
```

Comment:
I think we need to do the volume_tracking publishing here as well after an update and the same comment as above for library aliquots used in pool.